code/modules/camera/camera.dm

```diff
@@ -324,150 +321,64 @@
 		user.visible_message(SPAN_ALERT("[user] has reactivated [src]!"), SPAN_ALERT("You have reactivated [src]."))
 		add_fingerprint(user)
 
-/obj/machinery/camera/proc/autoposition()
-	// This will eventually be removed after wall cameras are repathed using an UpdatePaths script.
-	switch (src.dir)
-		if (NORTH, SOUTH)
-			src.pixel_x -= 10
-
-		if (EAST, WEST)
-			src.pixel_y += 10
-
-		if (NORTHWEST)
-			src.pixel_y -= 10
-			src.set_dir(WEST)
+/// Adds the minimap component for the camera
+/obj/machinery/camera/proc/add_to_minimap()
+	src.AddComponent(/datum/component/minimap_marker/minimap, src.minimap_types, "camera", name=src.c_tag)
 
-		if (NORTHEAST)
-			src.pixel_y -= 10
-			src.set_dir(EAST)
+SET_UP_DIRECTIONALS(/obj/machinery/camera, OFFSETS_CAMERA)
 
-		if (SOUTHEAST)
-			src.pixel_x += 10
-			src.set_dir(SOUTH)
+/obj/machinery/camera/cargo
+	name = "autoname - cargo"
```

Comment:
Same with these, we want spawned cameras to just be called "security camera"

Reply:
For routing cameras, it would be more accurate for them to be named as such, as they're on a separate network to security cameras.